styles/lib/style1.css

```diff
@@ -0,0 +1,157 @@
+
+*{padding:0px;
+    margin:0px;}
+
+
+body{
+
+    background-image: url('../../images/spezie.jpg');
+    width:100%;
+    height:100%;
+}
+
+header{
+    height:5%;
+    width:73%;
+    margin:0% auto;
+    background-color:darkred;
+}
+.number{
+
+    width:73%;
+    margin-left:63%;
+    color:navajowhite;
+    display:inline;
+    text-align:right;
+    font-family:"Helvetica Neue", Helvetica, Arial, sans-serif;
+    font-style:oblique;
+}
+
+.icon{
+    height:2%;
+    width:3%;
+    padding:1px;
+}
+
+.icon:hover{
+    border-radius:18px;
+}
+
+.company{
+    margin:0% auto;
+    width:73%;
+    height:7%;
+    background-color:navajowhite;
+    opacity:0.9;
+
+}
+
+
+.heading{
+    margin:1% auto;
+    width:73%;
+    font-family:cursive;
+    font-size:400%;
+    display:block;
+    margin-bottom:20px;
+    color:darkred;
+    padding:3px;
+    margin-left:57px;
+}
+
+label{
+    color:darkred;
+    font-size:15px ;
```

Comment:
font sizes should not be hardcoded, it should come from the root element, use em, rem or % in case you want to update the font size